packages/core/http/core-http-server/src/router/response.test.ts

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { isKibanaResponse } from './response';
+
+describe('isKibanaResponse', () => {
+  it('expects the status to be a number', () => {
+    expect(
+      isKibanaResponse({
+        status: 200,
+        options: {},
```

Comment:
The issue having this exposed from the public type package is that we can't even test it against our concrete implementation 😅.

However it's probably fine for now, I don't have a better suggestion / place to put it that wouldn't imply create yet another package, so if @jloleysens is fine with the approach, I'm too.